ci/scripts/connect-release-init.js

```diff
@@ -46,7 +48,7 @@ const initConnectRelease = () => {
     const errors = checkResult.errors.map(package => package.replace('@trezor/', ''));
 
     if (update) {
-        for (let packageName of update) {
+        update.forEach(packageName => {
```

Comment:
I wanted to run update for each package one by one synchronously. is forEach good idea here?

Reply:
I see, what about making everything asynchronous?  :pray: 

I really prefer to embrace the asynchronous nature of javascript since using synchronous method can cause unexpected issues.

Reply:
foreach is probably good here since everything in its callback is sync. Here, in release scripts, I believe having everything synchronous is better, safer, easier to read. We don't need any performance optimizations here